### ISSUE TITLE
GGMax Enhancement - Simple PASS_WORD STRING behaviour

### ISSUE THREAD

**Issue by firstbastion** [TheGameCreators/GameGuruRepo#5833]
Could use a simple STRING pass_word Switch  that would attach to a object.

Player types a 5-6 letter word.  If exact match correct - activates unlocks, sound file plays.  
if incorrect - failure message displays on local. 

The behaviour can have a few variables. 

A unique string field:  alpha numneric  whatever the actual password is.  example - dr3ary  

A proximity prompt telling the player to  Type pass_word
The match test is on the full word.

A message indicating failed attempt. 
A failure sound
A success sound

bonus:  Failure count  after 3 attempts the terminal shuts down.  

Please DO NOT make it like the "Code Switch" with each number individually tested.

**Comment by mav3r1ck198-1** [comment]
Can this not be done with the hack behavior? Or is that just for numbers

**Comment by Necrym59** [comment]
Yes he wants a password version of the hack behavior, i will get around to it when able to.
